modules/security-context-constraints-psa-rectifying.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-rectifying_{context}"]
+= Resolving permission errors that result from pod security admission synchronization
+
+TODO: *What to say here?*
```

Comment:
@stlaz Just starting to generate drafts for preview here, but I am keen on figuring out what to populate an error/warning resolution section with. Any thoughts?

Reply:
Or if these warnings are sufficiently helpful as they are, I can just remove this.

Reply:
@stlaz Anything you'd want to see here?

Reply:
We need to tell the users how to find the warnings in the audit log at least. From there it should be quite straightforward.

Reply:
Point them to the correct audit logs (kube-apiserver), tell them how to find events that show the policy violations (they have specific PSa label)